docs/flocker-features/flockerctl.rst

```diff
@@ -12,18 +12,13 @@ The ``flockerctl`` Command
         <p>This page describes one of our experimental projects, developed to less rigorous quality and testing standards than the mainline Flocker distribution. It is not built with production-readiness in mind.</p>
 	</div>
 
-``flockerctl`` is a CLI for controlling the Flocker Control Service.
-It is specific to datasets, and should not be confused with ``flocker-deploy`` (which is deprecated) or ``flocker-ca`` (which can be used as part of the manual initial configuration of a cluster's certificates).
-
-``flockerctl`` has commands for listing nodes, creating volumes, and moving them around.
-It supports supplying metadata such as size, name and :ref:`storage-profiles`.
+``flockerctl`` is a CLI for controlling the Flocker Control Service, with commands for listing nodes, creating volumes, and moving them around.
+``flockerctl`` supports supplying metadata such as size, name and :ref:`storage-profiles`.
 
 Install & Configure
 ===================
 
-``flockerctl`` is packaged along with the :ref:`labs-installer`.
-You don't have to use the labs installer in order to use ``flockerctl``, though.
-You can use ``flockerctl`` with Flocker installed via :ref:`any installation mechanism <supported-orchestration-frameworks>`.
+You can use ``flockerctl`` with Flocker installed via :ref:`any installation method <supported-orchestration-frameworks>`.
```

Comment:
:question: I'd be inclined to remove this line. There's no reason for a user to think that the tool might only work with certain installations.